### PR TITLE
Update dracut parse kickstart tests

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: tests/nosetests/dracut_tests/parse-kickstart_test.py [hunk 1]
```diff
@@ -129,11 +129,12 @@ def driverdisk_test_2(self):
 
     def network_test(self):
         with tempfile.NamedTemporaryFile(mode="w+t") as ks_file:
-            ks_file.write("""network --device=link --bootproto=dhcp --activate""")
+            ks_file.write("""network --device=ens3 --bootproto=dhcp --activate""")
             ks_file.flush()
             lines = self.execParseKickstart(ks_file.name)
 
-            self.assertRegex(lines[0], r"ip=[^\s:]+:dhcp: bootdev=[^\s:]+", lines)
+            print(lines)
```
[the remark this answers is not in the document]
Comment (jkonecny12) [reply tests/nosetests/dracut_tests/parse-kickstart_test.py:136]: Yes, you can. You should see stdout and stderr when the test fails.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me!

**Pull request merged**